src/test/java/org/jboss/xavier/integrations/EndToEndTest.java

```diff
@@ -827,6 +846,9 @@ public void configure() {
         assertThat(initialSavingsEstimationReportService.findByAnalysisOwnerAndAnalysisId("dummy@redhat.com", 3L)).isNull();
         assertThat(getStorageObjectsSize()).isEqualTo(s3ObjectsBefore - 1);
 
+        ResponseEntity<PagedResources<AnalysisModel>> responseAnalysisModel = new RestTemplate().exchange(getBaseURLAPIPath() + "/report?limit=2&offset=0", HttpMethod.GET, getRequestEntity(), new ParameterizedTypeReference<PagedResources<AnalysisModel>>() {});
```

Comment:
Could you add a comment to describe the test please? the rest of test always have something like:

```
// This test is for ....
```

Reply:
+1